shared/src/packages/sites/libs/enums/sites-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 const SitesApiPath = {
   ROOT: '/',
+  $PROJECT_ID: '/:projectId',
```

Comment:
```suggestion
```
the route should be `/projects/:projectId/sites` and it's a part of the `ProjectsApiPath `